code/game/gamemodes/cult/runes.dm

```diff
@@ -1,5 +1,6 @@
 /var/list/sacrificed = list() //a mixed list of minds and mobs
 var/list/non_revealed_runes = (subtypesof(/obj/effect/rune) - /obj/effect/rune/malformed)
+var/static/list/rune_types //Every rune that can be drawn by tomes
```

Comment:
Good to know. The DM ref doc didn't include anything as far as I could tell, so I was left to guess.